ci/authn-gcp/get_gcp_id_tokens.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+#	---------------------------------------------------------------------------------------------------------------------
+# This script obtains all of the GCP identity tokens required by authn-gcp cucumber tests
+# The scripts accepts the GCE instance name as an argument.
+# The script does the following:
+# 1. Validates gcloud is installed (requires a GCP account)
+#   To install gcloud goto: https://cloud.google.com/sdk/docs, download and install the SDK by running:
+#   ./google-cloud-sdk/install.sh
+#   ./google-cloud-sdk/bin/gcloud init
+# 2. Verifies the GCE instance exists and is running
+# 3. Executes a ssh curl command and write the output token with the appropriate token name to '../ci/authn-gcp/tokens'.
+#	---------------------------------------------------------------------------------------------------------------------
+
+PROGNAME=$(basename "$0")
+INSTANCE_ZONE=""
+TOKENS_OUT_DIR_PATH=../ci/authn-gcp/tokens
+TOKEN_FILE_NAME_PREFIX=gcp_token_
+INSTANCE_EXISTS=0
+INSTANCE_RUNNING=0
+
+main() {
+  echo "-- ------------------------------------------ --"
+  echo "-- Generate Google Cloud GCP Identity tokens --"
+  echo "-- ------------------------------------------ --"
+
+  echo "-- Verifying 'gcloud' is installed..."
+  ensure_gcloud_is_installed
+  echo "-- Verifying GCE instance name..."
+  check_instance_name_arg "$1"
+  echo "-- Checking if GCE instance: '${INSTANCE_NAME}' exists..."
+  ensure_instance_exists_and_running "$INSTANCE_NAME"
+  echo "-- Deleting stale token files..."
+  rm -rf ${TOKENS_OUT_DIR_PATH}/${TOKEN_FILE_NAME_PREFIX}*
```

Comment:
what is the `*`?

Reply:
oh oh it all files that match that prefix

Reply:
Yes, it is the tokens folder.